dominion/routines.py

```diff
@@ -43,6 +43,10 @@
     'Debian 10 "Buster" (32-bit)': 'debian-buster-armhf',
 }
 
+VALID_CHROOT_NAMES = {
```

Comment:
@leocoan I think `VALID_CHROOT_NAMES` is not suitable name. Can you please replace 'CHROOT` with something more suitable.